### ISSUE TITLE
Speed issue on clusters

### ISSUE THREAD

**Issue by TimSlendebroek** [ProjectTorreyPines/FUSE.jl#754]
[act.json](https://github.com/user-attachments/files/17709525/act.json)
[dd.json](https://github.com/user-attachments/files/17709526/dd.json)
[ini.json](https://github.com/user-attachments/files/17709527/ini.json)
[log.txt](https://github.com/user-attachments/files/17709528/log.txt)
[timer.txt](https://github.com/user-attachments/files/17709529/timer.txt)

Note the timer.txt gives extremely slow speeds per case of up to 3000 seconds for a ```ActorWholeFacility```

This isnt' the case locally (I am running on a single thread to make comparison more fair)::
```
@time FUSE.init(dd,ini,act);
 19.637729 seconds (22.34 M allocations: 3.284 GiB, 2.27% gc time)
@time FUSE.ActorWholeFacility(dd,act);
213.853568 seconds (763.70 M allocations: 44.551 GiB, 4.20% gc time, 44.83% compilation time: 1% of which was recompilation)
```

On saga
WholeFacility              2    1916s   88.4%    958s   23.4GiB   88.2%  11.7GiB

**Comment by orso82** [comment]
@TimSlendebroek I am wondering if we are somehow oversubscribing the nodes. Can you try to run with a very small population (say 4 individuals) and see what happens? You should see only four processes on a single node. How do the times compare?

**Comment by TimSlendebroek** [comment]
Yeah that's a good idea let me try a couple of variations and different cpus_per_task

**Comment by TimSlendebroek** [comment]
@orso82 

I tried running with much less workers than there are on a single node still giving the same issue

In addition to that I see good speeds when I requrest a worker node, ssh into that worker node and startup a simulation from the supposedly slow ini,act :(

**Comment by orso82** [comment]
maybe a Julia version issue then?

**Comment by TimSlendebroek** [comment]
This is no longer an issue it also helps precompiling everything not for specific hardware in a different enviornment
